dataproc/init_scripts/install_common.sh

```diff
@@ -14,30 +14,12 @@ fi
 conda install -c conda-forge mamba
 # Can't update Python 3.10 without removing pinned versions first.
 rm /opt/conda/miniconda3/conda-meta/pinned
-# Remove a few packages that prevent installing Python 3.10.
-mamba remove bcolz fiona pyqt spyder
 # Finally, update Python.
 mamba install python=3.10
```

Comment:
If we're now based on 22.04, what is the pre-existing python version?

AKA where does `conda` on line 14 come from.

It's fine as is, and I'm happy with your approach of minimal changes, but I'm just trying to convince myself that the `pip3` below does still correspond to the py3.10 being installed here…

Reply:
We'll chat about this on the call, but it comes from dataproc itself, not the dataproc driver image.

Reply:
Not related to this comment exactly, but it did lead us to find the image-version of the dataproc image is `2.1.2-debian11`, which comes with Python 3.10.8 by default, which means we've been able to simplify the init script to not having to update python.